package.json

```diff
@@ -1,23 +1,29 @@
 {
-    "private": false,
+    "private": true,
     "name": "@wso2is/webapps",
     "version": "0.1.0",
     "description": "WSO2 Identity Server Web Applications",
     "repository": "https://github.com/wso2/identity-apps.git",
     "scripts": {
-        "prebuild": "npm i && lerna bootstrap",
+        "prebuild": "yarn",
```

Comment:
Are we using `yarn` workspaces?

Reply:
Yup, this to avoid node_modules duplication